charts/nfs-subdir-external-provisioner/Chart.yaml

```diff
@@ -1,10 +1,10 @@
 apiVersion: v1
-appVersion: 3.1.0
+appVersion: 4.0.0
 description: nfs-subdir-external-provisioner is an automatic provisioner that used your *already configured* NFS server, automatically creating Persistent Volumes.
 name: nfs-subdir-external-provisioner
 home: https://github.com/kubernetes-sigs/nfs-subdir-external-provisioner
-version: 3.0.0
-kubeVersion: ">=1.9.0-0 <1.20.0-0"
+version: 4.0.0
+kubeVersion: ">=1.9.0-0"
```

Comment:
@yonatankahana => Is this a good time to change this?

Reply:
yes